src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -782,15 +790,20 @@ where
                     let into_ready_upper = timestamper
                         .reclock_frontier(ready_upper.borrow())
                         .expect("uninitialized reclock follower");
-                    trace!(
-                        "timely-{worker_id} reclock({id}) downgrading timestamper: since={}",
-                        into_ready_upper.pretty()
-                    );
 
                     cap_set.downgrade(into_ready_upper.elements());
                     timestamper.compact(into_ready_upper.clone());
                     if into_ready_upper.is_empty() {
+                        info!(
+                            "timely-{worker_id} reclock({id}) downgrading timestamper: since={}",
```

Comment:
```suggestion
                        info!(
                            %id,
                            "timely-{worker_id} reclock({id}) downgrading timestamper: since={}",
```